src/main/resources/2.7.x.conf

```diff
@@ -9,7 +9,7 @@ templatecontrol {
       { path = "/NOTICE", template = "NOTICE" },
       { path = "/.mergify.yml", template = ".mergify.yml" },
       { path = "/.github/settings.yml", template = "settings.yml" },
-      # { path = "/.travis.yml", template = ".travis.yml" }
+      { path = "/.travis.yml", template = ".travis.yml" }
```

Comment:
I thought `.travis.yml` was commented because some projects need a specific `.travis.yml`. Can't remember which but https://github.com/playframework/play-scala-secure-session-example seems to need custom bits:

https://github.com/playframework/play-scala-secure-session-example/blob/2.7.x/.travis.yml#L20-L22

Reply:
That's correct, and that's what this PR works itself into resolving.  See https://github.com/lightbend/templatecontrol/pull/121/commits/9df64b5230bb20dd2df848b9d5400548daab1f76 (which was the last commit until I had to resolve a dependency resolution problem on Travis CI)